livelog/extractor/writer.go

```diff
@@ -0,0 +1,56 @@
+package extractor
+
+import (
+	"bytes"
+	"encoding/base64"
+	"io"
+	"regexp"
+)
+
+var (
+	prefix = []byte("\u001B]1338;")
+	suffix = []byte("\u001B]0m")
+	re     = regexp.MustCompilePOSIX("\u001B]1338;((.*?)\u001B]0m)")
+)
+
+type Writer struct {
+	base    io.Writer
+	file    []byte
+	chunked bool
+}
+
+func New(w io.Writer) *Writer {
+	return &Writer{w, nil, false}
+}
+
+func (e *Writer) Write(p []byte) (n int, err error) {
```

Comment:
add feature flag to allow disabling cards (just in case there is an issue)

```diff
func (e *Writer) Write(p []byte) (n int, err error) {
+  if disableCards {
+    return e.base.Write(p)
+  }
```

Reply:
done.